audio/libopenmpt/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             libopenmpt
+version          0.6.7
+revision         0
+categories       audio devel multimedia
+platforms        darwin
+license          BSD
+maintainers      nomaintainer
+description      Library to decode tracked music files
+long_description ${name} is a cross-platform C++ and C library to decode \
+    tracked music files (modules) into a raw PCM audio stream.
+homepage         https://libsndfile.github.io/libsndfile/
+
+master_sites     https://lib.openmpt.org/files/libopenmpt/src
+distname         ${name}-${version}+release.autotools
+checksums        rmd160 edcaf6d9f0ea2cf3370423edd79afbdbbe2e0350 \
+    sha256 2174ac0f5a148ba684db768a47edf783eff9084fbca5fef6c997501643100163 \
+    size   1556248
+livecheck.type   regex
+livecheck.url    https://lib.openmpt.org/libopenmpt/
+livecheck.regex  "libopenmpt releases (\\d+\\.\\d+\\.\\d+),"
+
+depends_build    port:pkgconfig
+depends_lib      port:flac port:libogg port:libsndfile port:libvorbis \
+    port:mpg123 port:portaudio port:zlib
+
+configure.args \
+    --disable-debug \
+    --disable-dependency-tracking \
+    --disable-silent-rules \
+    --without-vorbisfile
```

Comment:
`--without-vorbisfile` is highly likely wrong here. libopenmpt requires libogg, libvorbis, and libvorbisfile to properly load OggVorbis compressed samples in `OXM` and `MO3` formats.

libvorbisfile (and libvorbisenc) are normally provided by the same libvorbis package as libvorbis itself, so I think it should be available in MacPorts just fine (looking at <https://github.com/macports/macports-ports/blob/master/audio/libvorbis/Portfile>).

I am not using MacPorts myself, so I do not have a quick way to verify either situation right now.

I suggest you take a second look at that.
If you are seeing compilation errors with libvorbisfile, I will happily take a look and fix any issues upstream as soon as possible.